index.html

```diff
@@ -9,7 +9,7 @@
       name="description"
       content="Author: Amedzro Emmanuel Elikplim, School: Microverse, Project: Portfolio"
     />
-
+    <link rel="stylesheet" href="menu.css" />
     <link rel="stylesheet" href="styles.css" />
 
     <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
Your "Roboto" fonts are not showing up even if you are declaring them on your CSS"

![image](https://user-images.githubusercontent.com/57421823/152019115-a592c6fc-9185-4159-a356-62b354e5ed11.png)

Upon checking the HTML, I can't see the "Roboto" fonts imported along with it's proper weights (I can see the Inter though).

- [ ] Please kindly import the "Roboto" fonts to your HTML along with the required font-weights so they actually show up on the page.  Don't forget to import the font weights that you need.

That way, your fonts will actually render on the page 💪

Reply:
Addressed 🔥